common/src/main/webapp/less/variables-light.less

```diff
@@ -62,6 +71,20 @@
   --scrollbar-background: lighten(@lightGrey, 5%);
   --scrollbar-background-thumb: darken(@lightGrey, 15%);
   --scrollbar-background-thumb-hover: darken(@lightGrey, 20%);
+  --hud-color: hsla(
```

Comment:
Is it detecting when a variable is defined in dark but not light?

Reply:
It can only know that if the color is being used in the scss files being linted. This variable is used in `vizualization.scss`, so `lint-light` caught the omission.

Reply:
Unfortunately, since we are no longer linting `less` files, we will no longer know that for `styles.less`. 🤷

Reply:
ok but still that seems very good